CHANGELOG.next.asciidoc

```diff
@@ -1059,6 +1059,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add Audit and Authentication Polixy Change Events and related.ip information {pull}20684[20684]
 - Add new ECS 1.8 improvements. {pull}23563[23563]
 - Remove deprecated eventlogging api that was used for Windows XP/2003 and associated unused code. {pull}24463[24463]
+- Move SafeReader to libbeat/common/encoding/xml. {pull}24560[24560]
```

Comment:
These don't need changelog entries since they don't affect end users. If it's a change that might affect other developers (like an API change affecting community beats) then you can put something in the developer changelog file.